### PR TITLE
Quests, Underworld achievements, Sanctum of Rebirth

### PR DESCRIPTION
**Pull request by SnowyToo**
Also includes misc fixes to make the new and some old quests work properly

### CODE REVIEW

File: rs3-chunkpicker-chunkinfo-export.json [hunk 1]
```diff
@@ -47336,6 +47350,22 @@
 				"NoPet": true,
 				"NoBoost": true
 			},
+			"Wield a ~|Roar of Awakening|~": {
+				"Items": ["Ode of Awakening"],
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:47354]: Should be "Items": ["Roar of Awakening"]

File: rs3-chunkpicker-chunkinfo-export.json [hunk 2]
```diff
@@ -59604,6 +59676,7 @@
 				"Not Equip": true,
 				"Priority": 1
 			},
+
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: typo I think?

**Review by ThaFrenchFry — CHANGES_REQUESTED** [review]
Fix the roar and the extra space and you're golden

**Review by ThaFrenchFry — APPROVED** [review]

**Review by Connor3455 — APPROVED** [review]

**Comment by source-chunk** [comment]
Implemented via #2

**Pull request closed without merging**